### PR TITLE
chore: add go cover in makefile

### PR DESCRIPTION
**Pull request by shubham-cmyk**
## Explanation

Add makefile target to get the code coverage

## Checklist

- [x] I have read the [contributing guidelines](https://github.com/kyverno/kyverno/blob/main/CONTRIBUTING.md).
- [x] I have read the [PR documentation guide](https://github.com/kyverno/kyverno/blob/main/.github/pr_documentation.md) and followed the process including adding proof manifests to this PR.
- [ ] This is a bug fix and I have added unit tests that prove my fix is effective.

### CODE REVIEW

**Comment by eddycharly** [comment]
Why not add it by default to the existing target ?

```
.PHONY: tests
tests: $(CLI_BIN) ## Run tests
	@echo Running tests... >&2
	@go test ./... -race -coverprofile=coverage.out -covermode=atomic
	@go tool cover...
```

**Comment by shubham-cmyk** [comment]
@eddycharly sure why not

File: Makefile [hunk 1]
```diff
@@ -211,6 +211,7 @@ build: $(CLI_BIN) ## Build
 tests: $(CLI_BIN) ## Run tests
 	@echo Running tests... >&2
 	@go test ./... -race -coverprofile=coverage.out -covermode=atomic
+	@go tool cover -html=coverage.out
```
Comment (eddycharly) [inline Makefile:214]: I think you will need to output to a file

Comment (shubham-cmyk) [reply Makefile:214]: yeah if i don't output to a file it open up in the browser.

Comment (eddycharly) [reply Makefile:214]: yup, looks like ci is green though 🤷

Comment (shubham-cmyk) [reply Makefile:214]: yes. but Now I have pushed the change LOL

Comment (shubham-cmyk) [reply Makefile:214]: not pushed this can be merged

**Review by eddycharly — APPROVED** [review]

**Pull request merged**